tests/launch-mock-event-recorder-tests.sh

```diff
@@ -31,13 +31,13 @@ export DBUS_SESSION_BUS_PID
 trap finally EXIT
 
 # Start the mock service and add its methods.
-python -m dbusmock --system \
+python3 -m dbusmock --system \
```

Comment:
Not sure what the best way is here to use the detected Python executable. Consider turning this into `launch-mock-event-recorder-tests.sh.in`, using `@PYTHON@`, and listing this file in `AC_CONFIG_FILES`, combined with `AC_SUBST([PYTHON])` in `configure.ac`.